### PR TITLE
[installer] Replace flag withoutWorkspaceComponents with 'Kind != "Full"'

### PR DESCRIPTION
**Pull request by geropl**
## Related Issue(s)
Context: https://github.com/gitpod-io/gitpod/pull/15276

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: install/installer/pkg/config/v1/experimental/experimental.go [hunk 1]
```diff
@@ -205,21 +205,20 @@ type SpiceDBConfig struct {
...
-	Tracing                    *Tracing               `json:"tracing,omitempty"`
-	UsePodAntiAffinity         bool                   `json:"usePodAntiAffinity"`
-	DisableMigration           bool                   `json:"disableMigration"`
-	Usage                      *UsageConfig           `json:"usage,omitempty"`
-	ConfigcatKey               string                 `json:"configcatKey"`
-	WorkspaceClasses           []WebAppWorkspaceClass `json:"workspaceClasses"`
-	Stripe                     *StripeConfig          `json:"stripe,omitempty"`
-	SlowDatabase               string                 `json:"slowDatabase,omitempty"`
-	IAM                        *IAMConfig             `json:"iam,omitempty"`
-	WithoutWorkspaceComponents bool                   `json:"withoutWorkspaceComponents,omitempty"`
```
Comment (easyCZ) [inline install/installer/pkg/config/v1/experimental/experimental.go:221]: Diff is weird, but for future readers this bool field is removed.

**Review by easyCZ — APPROVED** [review]

**Review by WVerlaek — APPROVED** [review]
lgtm

**Review by vulkoingim — APPROVED** [review]

File: install/installer/pkg/config/v1/experimental/experimental.go (hunk 1 above)
Comment (corneliusludmann) [reply install/installer/pkg/config/v1/experimental/experimental.go:221]: Hide whitespace gives you a proper diff view.

image

image

**Pull request merged**